### PR TITLE
Adding a multi drag pattern

### PR DESCRIPTION
**Pull request by alexreardon**

### CODE REVIEW

File: stories/src/multi-drag/reorder.old.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
+// @flow
```
Comment (Blasz) [inline stories/src/multi-drag/reorder.old.js]: I can't see this file being imported anywhere, is it being used?

File: stories/src/multi-drag/reorder.old.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  columns,
+  selected,
+  source,
+  destination,
+}): Result => {
+  // 1. remove all of the selected tasks from their lists
+  // When ordering the collected tasks:
+  //  dragged item first
+  //  followed by the items with the lowest index
+  //  in the event of a tie, use the one that was selected first
```
Comment (Blasz) [inline stories/src/multi-drag/reorder.old.js]: Should this order comment be located closer to the `collection.sort` line?

File: stories/src/multi-drag/reorder.old.js [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+  tasks,
+});
+
+const reorderMultiDrag = ({
+  columns,
+  selected,
+  source,
+  destination,
+}): Result => {
+  // 1. remove all of the selected tasks from their lists
```
Comment (Blasz) [inline stories/src/multi-drag/reorder.old.js]: This is the only numbered comment in the function. Should we remove the number or add some to the other comments?

File: stories/src/multi-drag/utils.js [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+    columns: withAddedTasks,
+  };
+
+  return {
+    entities: updated,
+    selectedTaskIds: orderedSelectedTaskIds,
+  };
+};
+
+export const reorderBetweenColumns = (args: Args): Result => {
```
Comment (Blasz) [inline stories/src/multi-drag/utils.js]: Doesn't this function reorder tasks in the same column as well? Is there a better name for this?

Comment (alexreardon) [reply stories/src/multi-drag/utils.js]: Hmm!

Comment (alexreardon) [reply stories/src/multi-drag/utils.js]: `mutliDragAwareReorder` seems alright for now

**Comment by LuoweiC** [comment]
I want to use this,but i can't use，can you give me some guidance?

**Comment by Frankcarvajal** [comment]
@LuoweiC What specifically about it can you not use. May you provide some specific issues you are dealing with?

I've been able to recreate a multi drag supported task list in my own open source repository by using the src code story from here if that may assist I can link it.

**Comment by geobde** [comment]
@Frankcarvajal can you provide an example?

**Comment by Frankcarvajal** [comment]
@geobde I've got an example repository on my GitHub, but it doesn't have an example of a multi drag and drop table, just standard like an example. @alexreardon has no multi drag and drop table example and the issue I put in for this assigned to me, I've not had a moment to work on. I do wish he would put an example of it up

**Pull request merged**